airbyte-cdk/python/airbyte_cdk/sources/declarative/models/declarative_component_schema.py

```diff
@@ -15,12 +11,12 @@
 
 
 class AddedFieldDefinition(BaseModel):
-    type: Literal["AddedFieldDefinition"]
+    type: Literal['AddedFieldDefinition']
     path: List[str] = Field(
         ...,
-        description="List of strings defining the path where to add the value on the record.",
-        examples=[["segment_id"], ["metadata", "segment_id"]],
-        title="Path",
+        description='List of strings defining the path where to add the value on the record.',
+        examples=[['segment_id'], ['metadata', 'segment_id']],
+        title='Path',
```

Comment:
This file is preferring single quotes, while others later in the PR are preferring double quotes. Might be worth investigating what is driving the different behavior.

I think general recommended default with Blank is to prefer double quotes except when single quotes are cleaner. 

https://black.readthedocs.io/en/stable/the_black_code_style/current_style.html#strings

Comparison file later in the PR:

`resources/examples/airflow/dags/dag_airbyte_example.py`

Reply:
This is resolved in the subsequent commit: 

https://github.com/airbytehq/airbyte/pull/29798/commits/db27f271b2b7a8114e125bca8dcd50d3dea952aa

Reply:
As it turns out the issue was a bit more complicated and involved code generation. It's since been fixed. Furthermore, I now have the python license task run globally as well.